static/app/data/experimentConfig.tsx

```diff
@@ -14,19 +14,7 @@ export const unassignedValue = -1;
  */
 export const experimentList = [
   {
-    key: 'IntegrationDirectoryCategoryExperiment',
-    type: ExperimentType.Organization,
-    parameter: 'variant',
-    assignments: ['0', '1'],
-  },
-  {
-    key: 'TrialEndingNotice',
```

Comment:
These are both already removed?

Reply:
@EvanPurkhiser Yep, it's not in the list of experiments anymore